src/main/java/com/firebolt/jdbc/connection/FireboltConnection.java

```diff
@@ -514,87 +528,75 @@ public void setHoldability(int holdability) throws SQLException {
 	}
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
 	public Savepoint setSavepoint() throws SQLException {
 		// No support for transaction
 		throw new FireboltSQLFeatureNotSupportedException();
 	}
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
 	public Savepoint setSavepoint(String name) throws SQLException {
 		// No support for transaction
 		throw new FireboltSQLFeatureNotSupportedException();
 	}
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
 	public void rollback(Savepoint savepoint) throws SQLException {
 		// No support for transaction
 		throw new FireboltSQLFeatureNotSupportedException();
 	}
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
 	public void releaseSavepoint(Savepoint savepoint) throws SQLException {
 		throw new FireboltSQLFeatureNotSupportedException();
 	}
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
 	public CallableStatement prepareCall(String sql, int resultSetType, int resultSetConcurrency,
 			int resultSetHoldability) throws SQLException {
 		throw new FireboltSQLFeatureNotSupportedException();
 	}
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
 	public Clob createClob() throws SQLException {
 		throw new SQLFeatureNotSupportedException();
 	}
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
 	public Blob createBlob() throws SQLException {
 		throw new SQLFeatureNotSupportedException();
 	}
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
 	public NClob createNClob() throws SQLException {
 		throw new SQLFeatureNotSupportedException();
 	}
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
```

Comment:
Can you expand on what is this doing and why is it being removed?

Reply:
This annotation excludes the code of the method from the test coverage report. I have added test that validates this method, so the annotation can be removed.